tests/garage/np/algos/test_cem.py

```diff
@@ -21,8 +21,6 @@ def test_cem_cartpole(self):
 
             baseline = LinearFeatureBaseline(env_spec=env.spec)
 
-            runner.initialize_tf_vars()
```

Comment:
We have already provided policies with tf session by runner, but looks like sometimes they want more. @ahtsan Do you think this problem will recur in the future? If so, we may need a mechanism more fundamental.

Reply:
I think it is fine for now. The runner experience is pretty ok for single task RL.